DHL/Client/Web.php

```diff
@@ -54,12 +54,19 @@ class Web
      * 
      * @var string $mode Mode can be either production or staging
      */
-    public function __construct($mode = 'staging')
+    public function __construct($mode = 'staging', $enableUtf8Support = true)
     {
         if (!in_array($mode, array('staging', 'production'))) 
         {
             throw new \InvalidArgumentException('Invalid mode : ' . $mode . '. Accepted values are : staging or production.');
         }
+
+        if ($enableUtf8Support)
+        {
+            $this->_stagingUrl .=  "?isUTF8Support=true";
+            $this->_productionUrl .=  "?isUTF8Support=true";
```

Comment:
Good. When you have local copy/paste scenarios like this, you can alternatively write:
```php
            $this->_stagingUrl .=  ($str = "?isUTF8Support=true");
            $this->_productionUrl .=  $str;
```

The parenthetical enclosure is unnecessary as assignment operators (`=`, `.=`) are *right-associative*, meaning that the order of precedence goes from the right to the left. I include parentheses because it increases readability and does not rely on another programmer's level of familiarity with PHP's order of operations. The PHP Manual itself makes this point:

> Use of parentheses, even when not strictly necessary, can often increase readability of the code by making grouping explicit rather than relying on the implicit operator precedence and associativity.

I highly recommend reading the source page from the PHP manual:

https://www.php.net/manual/en/language.operators.precedence.php